index.bs

```diff
@@ -4040,11 +4040,11 @@ partial dictionary MLOpSupportLimits {
 <dl dfn-type=dict-member dfn-for=MLGruOptions>
     : <dfn>bias</dfn>
     ::
-        The 2-D input bias tensor of shape *[numDirections, 3 * hiddenSize]*. The ordering of the bias vectors in the second dimension of the tensor shape is specified according to the {{MLGruOptions/layout}} argument.
+        The 2-D input bias tensor of shape *[numDirections, 3 * hiddenSize]*. The ordering of the bias vectors in the second dimension of the tensor shape is specified according to {{MLGruOptions/layout}}.
```

Comment:
Should we format each `numDirections` and `hiddenSize` separately? Like `[*numDirections*, 3 * *hiddenSize*]`, same for other occurrences.

Reply:
Right now the [coding conventions](https://github.com/webmachinelearning/webnn/blob/main/docs/SpecCodingConventions.md) say `When concisely defining a tensor's layout, use the syntax *[ ... ]* ` which we'd need to revise. More feedback wanted!